tests/wlr_virtual_pointer_v1.cpp

```diff
@@ -275,6 +292,7 @@ TEST_F(VirtualPointerV1Test, if_frame_is_not_sent_client_sees_no_events)
     zwlr_virtual_pointer_v1_axis_source(handle, WL_POINTER_AXIS_SOURCE_WHEEL);
     // Should produce no events because there has been no frame
```

Comment:
This test should actually *test* that no events are produced. As it is, I'm pretty sure this test would pass (but might log some warnings) even if events were generated?

This is pre-existing, though.